syntax = "proto3";

package giveaway;

service Greeter {
    rpc GetGiveaway (GiveawayRequest) returns (GiveawayReply) {}
    rpc ParticipatingGiveaway (GiveawayRequest) returns (GiveawayReply) {}
}

message GiveawayRequest {
    string initData = 1;
}

message GiveawayReply {
    string json_message = 1;
}